<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigDialog</class>
 <widget class="QDialog" name="ConfigDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>181</width>
    <height>338</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Configurations</string>
  </property>
  <widget class="QLabel" name="cnfgUsernameLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Username</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="cnfgUsernameEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>141</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="cnfgPortLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>66</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Port</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="cnfgPortEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>141</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="cnfgServerIPLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>66</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Server IP</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="cnfgServerIPEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>141</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="cnfgOkButton">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>200</y>
     <width>81</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>141</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>Colour:</string>
   </property>
   <widget class="QRadioButton" name="blueRadio">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>51</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>blue</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="greenRadio">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>20</y>
      <width>61</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>green</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="redRadio">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>50</y>
      <width>51</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>red</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="brownRadio">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>50</y>
      <width>71</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>brown</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="cyanRadio">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>80</y>
      <width>51</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>cyan</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="magentaRadio">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>80</y>
      <width>81</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>magenta</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
